boneyardproj/urls.py

```diff
@@ -16,9 +16,12 @@
 from django.contrib import admin
 from django.urls import path
 from main_app import views
+from main_app import services
+from django.conf.urls import url
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('test/', views.home),
-    path('', views.index)
+    path('', views.index),
+    url(r'^api/(?P<pk>[0-9]+)', services.get_parks),
```

Comment:
I don't understand this part too much. I think the 'r'^api/...  is first the path (<website url>/api/) but the part after that is maybe a "allow for numbers or letters to come after". Not sure

Reply:
Good work here!